### DISCUSSION TITLE
Inconsistencies between MonitorControl and Hardware Brightness Percentages

### DISCUSSION THREAD

**Discussion by DavenMauff** [MonitorControl/MonitorControl#1377] [General]
It seems that the brightness percentages displayed by the MonitorControl app and the hardware control differ at various levels. Specifically, when I set MonitorControl to 100%, it matches 100% on the hardware. However, at 90% in MonitorControl, it shows 81% on the hardware, at 75% it shows as 52% on the hardware, and at 50% it shows as 1% on the hardware. Additionally, I've noticed that these percentages can vary inconsistently. For instance, setting it to 50% may sometimes result in 1% or even 0% on the hardware. I haven't made any adjustments to the advanced settings yet. I hope this issue can be addressed in a future update because I'm really enjoying the app.

I'm currently using a Dell Monitor model S2721DGFA as a point of reference.

**Comment by waydabber** [comment]
This is by design. By default the app uses software dimming between 0-50% and hardware between 50%-100% of the slider range. You can disable this behavior.
